banner

```diff
@@ -0,0 +1,120 @@
+diff --git a/www/src/components/navigation.js b/www/src/components/navigation.js
+index a368e03da..b9dcfdc6f 100644
+--- a/www/src/components/navigation.js
++++ b/www/src/components/navigation.js
+@@ -96,6 +96,7 @@ export default ({ pathname }) => {
+         zIndex: `2`,
+         left: 0,
+         right: 0,
++        top: `calc(${presets.bannerHeight} - 1px)`,
+         [presets.Tablet]: {
+           position: isHomepage || isBlog ? `absolute` : `fixed`,
+         },
+diff --git a/www/src/layouts/index.js b/www/src/layouts/index.js
+index f5e2a4784..9685e82c7 100644
+--- a/www/src/layouts/index.js
++++ b/www/src/layouts/index.js
+@@ -8,7 +8,8 @@ import SearchBar from "../components/searchbar-body"
+ import tutorialSidebar from "../pages/docs/tutorial-links.yml"
+ import docsSidebar from "../pages/docs/doc-links.yaml"
+ import featuresSidebar from "../pages/docs/features-links.yaml"
+-import { rhythm } from "../utils/typography"
++import { OutboundLink } from "gatsby-plugin-google-analytics"
++import { rhythm, options, scale } from "../utils/typography"
+ import presets, { colors } from "../utils/presets"
+ import hex2rgba from "hex2rgba"
+ import "../css/prism-coy.css"
+@@ -60,9 +61,11 @@ class DefaultLayout extends React.Component {
+       width: rhythm(10),
+       display: `none`,
+       position: `fixed`,
+-      top: `calc(${presets.headerHeight} - 1px)`,
++      top: `calc(${presets.headerHeight} + ${presets.bannerHeight} - 1px)`,
+       overflowY: `auto`,
+-      height: `calc(100vh - ${presets.headerHeight} + 1px)`,
++      height: `calc(100vh - ${presets.headerHeight} - ${
++        presets.bannerHeight
++      } + 1px)`,
+       WebkitOverflowScrolling: `touch`,
+       "::-webkit-scrollbar": {
+         width: `6px`,
+@@ -132,14 +135,52 @@ class DefaultLayout extends React.Component {
+           />
+           <html lang="en" />
+         </Helmet>
++        <div
++          css={{
++            width: `100%`,
++            padding: rhythm(1 / 2),
++            background: isHomepage ? `#402060` : colors.gatsby,
++            color: colors.ui.bright,
++            fontFamily: options.headerFontFamily.join(`,`),
++            fontSize: scale(-1 / 5).fontSize,
++            zIndex: `3`,
++            [presets.Desktop]: {
++              position: `fixed`,
++            },
++            [presets.Tablet]: {
++              position: `fixed`,
++            },
++            WebkitFontSmoothing: `antialiased`,
++          }}
++        >
++          These are the docs for V1.{` `}
```

Comment:
My preference would be to make this more of a call to action since we want developers to try the V2 beta.

How about this text instead?

> Gatsby v2 is now in beta!

Reply:
Sure, This sound great